scrapy/commands/startproject.py

```diff
@@ -43,41 +44,68 @@ def _module_exists(module_name):
         if not re.search(r'^[_a-zA-Z]\w*$', project_name):
             print('Error: Project names must begin with a letter and contain'\
                     ' only\nletters, numbers and underscores')
-        elif exists(project_name):
-            print('Error: Directory %r already exists' % project_name)
         elif _module_exists(project_name):
             print('Error: Module %r already exists' % project_name)
         else:
             return True
         return False
 
+    def _copytree(self, src, dst):
```

Comment:
Also, do you think we could get rid of this function, if we use the `ignore` callable argument for `shutil.copytree` ?